lib/sources/udp_link.cpp

```diff
@@ -21,43 +21,44 @@ UdpLink::UdpLink(unsigned int localPort, const string& localAddress, unsigned in
 {
 }
 
-int UdpLink::open()
+std::error_code UdpLink::open()
```

Comment:
It is not a good practice to return error code in method, better to call errorCode() after bind/open/close

Reply:
Why is it so?